master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -167,8 +167,8 @@ func (a *agentResourceManager) Receive(ctx *actor.Context) error {
 				ctx.Log().WithError(aMsg).Error("")
 				ctx.Respond(aMsg)
```

Comment:
if there are multiple errors we will ctx.Respond multiple times, but a ctx.Respond closes the channel it sends on and sends on close channels panic, so this will crash the rm if a resource pool crashes.

Reply:
doesn't it return nil right after?

Reply:
1. i haven't had coffee yet, woke up 5m ago
2. you're up!?

duh, i can't read. but i do think it makes sense to accumulate the error.

Reply:
yeah why are you guys up?!!